### ISSUE TITLE
Set manual to manual.threefold.io instead of manual.grid.tf

### ISSUE THREAD

**Issue by Mik-TF** [threefoldtech/info_grid#684]
# Situation

- We want to simplify all the links to the main URLs of the ecosystem
- It would be nice to have manual.threefold.io, as we have forum.threefold.io, docs.threefold.io, etc.

# Notes

- The basic way is a redirect from manual.threefold.io to manual.grid.tf, but maybe we could do manual.threefold.io as the new main link (thus [update the workflow URL](https://github.com/threefoldtech/info_grid/blob/f694eef795afcda264f06f245034c7516b4825f7/.github/workflows/manual_update_master.yml#L1)), and update the dashboard + other links, and have a redirect of manual.grid.tf to manual.threefold.io to ease the transition, that would be better for the future and SEO

@xmonader wdyt?

**Comment by xmonader** [comment]
I don't agree witht that, for such changes there needs to be stakeholders approval.

**Comment by Mik-TF** [comment]
Waiting until we discuss with stakeholders. For now, we keep V3 process of using grid.tf as main for dashsboard and manual.
